tutorials/build_systems/easybuild/eb_test.py

```diff
@@ -0,0 +1,25 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class EasybuildMixin(rfm.RegressionTest):
+    @rfm.run_before('run')
+    def prepare_run(self):
+        self.modules = self.build_system.generated_modules
```

Comment:
I don't think that the mixin offers something in this example. I would suggest moving everything into the test class.